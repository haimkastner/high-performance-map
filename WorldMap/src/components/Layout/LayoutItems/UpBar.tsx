import { Box } from '@material-ui/core';
import Button from '@material-ui/core/Button';
import React from 'react';
export class UpBar extends React.Component {
  public render() {
    return (
      <Box>
        <Button>I am up button</Button>
      </Box>
    );
  }
}
