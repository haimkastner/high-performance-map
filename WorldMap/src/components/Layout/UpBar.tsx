import React from "react";
import styles from "./Styles/UpBar.module.css";
import { Box } from "@material-ui/core";
import Button from "@material-ui/core/Button";
export class UpBar extends React.Component {
  render() {
    return (
      <Box className={styles.UpBar}>
        <Button>I am up button</Button>
      </Box>
    );
  }
}
