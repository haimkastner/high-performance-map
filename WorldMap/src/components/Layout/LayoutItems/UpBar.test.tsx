import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { UpBar } from './UpBar';

describe('UpBar Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<UpBar />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('matches the snapshot', () => {
    const tree = renderer.create(<UpBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // tslint:disable-next-line: no-empty
  describe('UpBar Component Logic', () => {});
});
