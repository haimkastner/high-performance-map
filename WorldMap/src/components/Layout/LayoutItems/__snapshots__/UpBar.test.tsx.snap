// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpBar Component matches the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-36"
>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-text"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      I am up button
    </span>
  </button>
</div>
`;
